api/src/main/java/org/openmrs/module/htmltojson/htmltojson/HtmlFormUtil.java

```diff
@@ -145,6 +147,16 @@ public static ObjectNode getFormSchemaJson(String formUuid, ResourceFactory reso
 						} else {
 							
 							ObjectNode dataPointJson = generateJsonObjectForHtmlDataPoint(element);
+							if (currentIndex >= 3) {
+								int labelIndex = currentIndex - 2;
+								Element labelElement = elementTags.get(labelIndex);
+								String label = labelElement.toString().replace("<td class=\"lblcolor\">", "")
```

Comment:
Should we replace these before the json is generated?

Reply:
yes if that is possible, sure but as I said this is just an attempt am not sure if will work for all HFE files. If you can point where exactly to look at I can give it a try, I have not took the time to review the whole code base.